Default to enabling the ethernet interface. If we don't do this, connman
will shut down all the networking when it starts without any configuration
which is usualy not what the users expects. This is particularly
problematic with our qemu images and runtime testing using qemu.

Upstream-Status: Inappropriate [Configuration]

RP 2011/12/12

Index: git/src/storage.c
===================================================================
--- git.orig/src/storage.c	2011-12-12 16:14:10.892316565 +0000
+++ git/src/storage.c	2011-12-12 16:14:13.788316431 +0000
@@ -367,7 +367,7 @@
 					"Enable", FALSE);
 
 		g_key_file_set_boolean(keyfile, "Wired",
-					"Enable", FALSE);
+					"Enable", TRUE);
 
 		g_key_file_set_boolean(keyfile, "3G",
 					"Enable", FALSE);
